inspirehep/config.py

```diff
@@ -69,11 +69,11 @@ def _(x):
 
 # Celery
 # ======
-BROKER_URL = "amqp://guest:guest@localhost:5672//"
-CELERY_RESULT_BACKEND = "amqp://guest:guest@localhost:5672//"
+CELERY_BROKER_URL = "pyamqp://guest:guest@localhost:5672//"
+CELERY_RESULT_BACKEND = "rpc://guest:guest@localhost:5672//"
 CELERY_ACCEPT_CONTENT = ['json', 'msgpack', 'yaml']
 CELERY_TIMEZONE = 'Europe/Amsterdam'
-CELERY_DISABLE_RATE_LIMITS = True
+CELERY_WORKER_DISABLE_RATE_LIMITS = True
```

Comment:
Do we really need this? I'm going to benchmark it to see if it really improves performance, as was claimed in https://github.com/inspirehep/inspire-next/commit/95f1cfe03e6c4af6b52bab8f7e8a8d65148cee33.